front/src/stories/RollingStockSelector.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import RollingStockSelector from 'common/RollingStockSelector/RollingStockSelector';
+import { OSRD_ROLLINGSTOCKSELECTED_SAMPLE_DATA } from 'applications/operationalStudies/components/SimulationResults/SpeedSpaceChart/sampleData';
+import { RollingStock } from 'common/api/osrdEditoastApi';
+
+export default {
+  title: 'Common/RollingStockSelector',
+  component: RollingStockSelector,
+} as ComponentMeta<typeof RollingStockSelector>;
+
+const Template: ComponentStory<typeof RollingStockSelector> = (args) => (
+  <div className="w-50">
+    <RollingStockSelector {...args} />
+  </div>
+);
+
+export const NotCondensedRollingStockSelector = Template.bind({});
```

Comment:
Can you add a view of the empty RollingStockSelector ? :)

Reply:
done